cluster/ubuntu/config-default.sh

```diff
@@ -32,6 +32,21 @@ export NUM_NODES=${NUM_NODES:-3}
 # according to rfc 1918 ref: https://tools.ietf.org/html/rfc1918 choose a private ip range here.
 export SERVICE_CLUSTER_IP_RANGE=${SERVICE_CLUSTER_IP_RANGE:-192.168.3.0/24}  # formerly PORTAL_NET
 # define the IP range used for flannel overlay network, should not conflict with above SERVICE_CLUSTER_IP_RANGE
+
+# This script supports two ways of networking: Flannel and CNI.  To
+# use CNI: (1) put a CNI configuration file, whose basename is the
+# configured network type plus ".conf", somewhere on the driver
+# machine (the one running `kube-up.sh`) and set CNI_PLUGIN_CONF to a
+# pathname of that file, and (2) put one or more executable binaries
+# on the driver machine and set CNI_PLUGIN_EXES to a space-separated
+# list of their pathnames (relative to kubernetes/cluster).
+
+export CNI_PLUGIN_CONF=""
+export CNI_PLUGIN_EXES=""
```

Comment:
I'm expected to manually edit this file before I execute it to set CNI_PLUGIN_* ?